homeassistant/components/nws_alerts/config_flow.py

```diff
@@ -0,0 +1,144 @@
+"""Config flow for NWS Alerts integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import requests
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import API_ENDPOINT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Can you create the logger inside `const.py` as `LOGGER = logging.getLogger(__package__)` and import it here instead?

```suggestion
from .const import API_ENDPOINT, DOMAIN, LOGGER
```